File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int n = nums.size();
        int ptr1 = 0;
        for (int ptr2 = 1; ptr2<n ; ptr2++) {
            if ( nums[ptr1] != nums[ptr2] ) {
                nums[ptr1 + 1] = nums[ptr2];
                ptr1++ ;
            }
        }   
        return ptr1 + 1 ;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        n = len(nums)
        i = 0
        for j in range(1,n):
            if (nums[j]!=nums[i]) :
                nums[i+1] = nums[j]
                i+=1
        return i+1        
        